spec/views/catalog/librarian_view.html.erb_spec.rb

```diff
@@ -5,22 +5,24 @@
   include MarcMetadataFixtures
   describe "MARC records" do
     before do
-      assign(:document, SolrDocument.new(id: '12345', marcxml: metadata1))
+      assign(:document, SolrDocument.new(id: '12345', last_updated: '2022-08-01T23:01:18Z', marcxml: metadata1))
       render
     end
 
     it "should render the marc_view" do
+      expect(rendered).to have_content(l('2022-08-01T23:01:18Z'.to_time, format: :long))
```

Comment:
For clarity about our expectations, I'd suggest checking the actual text:

```suggestion
      expect(rendered).to have_content('August  1, 2022 11:01pm')
```

(no guarantees I got that right 😬 )